app/mailers/notification_mailer.rb

```diff
@@ -1,10 +1,23 @@
 class NotificationMailer < ActionMailer::Base
   default from: 'no-reply@gitreports.com'
 
-  def issue_submitted_email(repo_id, issue_id)
+  def issue_submitted_email(repo_id, issue_id, submitter_info = {})
+    defaults = {
```

Comment:
Lines 5-10 here shouldn't actually be necessary. `submitter_info` defaults to a hash so it will always be present, and down below `submitter_info[:submitter_name]` or w/e will do what you want, it doesn't need to be initialized to `nil`